django_project/geohosting/src/redux/reducers/productsSlice.ts

```diff
@@ -52,6 +52,23 @@ const initialState: ProductsState = {
   detailError: null,
 };
 
+const getCurrencyBasedOnLocation = async () => {
```

Comment:
Hi @tinashechiraya 
Could put this function to new utils file?
In this folder: https://github.com/kartoza/GeoHosting/tree/main/django_project/geohosting/src/utils
So we can pick it up faster

**localtion_helper.tsx** for example